### PR TITLE
 media queries completed

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-12-16 at 8 11 20 AM
Screen Shot 2021-12-16 at 8 11 12 AM
Kapture 2021-12-16 at 08 12 07
Kapture 2021-12-16 at 08 14 46

### CODE REVIEW

**Comment by dcatan89** [comment]
Kapture 2021-12-16 at 08 14 46

**Comment by dcatan89** [comment]
Kapture 2021-12-16 at 08 17 28

File: css-media-queries/breakpoints.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+      @media only screen
+  and (min-width: 767px)
+
+    {
+      .card-wrapper {
+        flex-basis: calc( 100%/ 3);
+      }
+    }
```
Comment (codingcodymiller) [inline css-media-queries/breakpoints.html]: Please fix your indentation.

File: css-media-queries/at-media.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+@media (min-width: 768px) {
+  body {
+    background-color: papayawhip;
+  }
+}
```
Comment (codingcodymiller) [inline css-media-queries/at-media.html]: Please fix your indentation.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-media-queries/at-media.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    /* your code here 👇 */
+@media only screen
+and (min-width: 768px) {
+
+  body {
+    background-color: papayawhip;
+  }
+}
```
Comment (codingcodymiller) [inline css-media-queries/at-media.html]: This is still not correct indentation.

File: css-media-queries/breakpoints.html (hunk 1 above)
Comment (codingcodymiller) [reply css-media-queries/breakpoints.html]: THis is still not correct indentation. Please follow the format taught in the documentation.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**